conjure-java-core/src/integrationInput/java/test/prefix/com/palantir/product/UnionWithUnknownString.java

```diff
@@ -38,6 +39,16 @@ public static UnionWithUnknownString unknown_(String value) {
         return new UnionWithUnknownString(new Unknown_Wrapper(value));
     }
 
+    public static UnionWithUnknownString unknown(String type, Object value) {
+        switch (Preconditions.checkNotNull(type, "Type is required")) {
+            case "unknown_":
```

Comment:
We should add a unit test for this -- looks like we're using the java variable name instead of the type value.